modin/test/test_partition_api.py

```diff
@@ -83,6 +83,24 @@ def get_df(lib, data):
                     get_func(expected_partitions[row_idx][col_idx].list_of_blocks[0]),
                     get_func(actual_partitions[row_idx][col_idx]),
                 )
+        df = get_df(pd, data)
```

Comment:
Let's move this to another test case, and I think there's no need to parametrize by `reverse_index`, `axis` or `reverse_columns`: we can just have axis of `None`.